javaparser-core/src/main/java/com/github/javaparser/ast/expr/MethodCallExpr.java

```diff
@@ -65,6 +65,14 @@ public MethodCallExpr(final Expression scope, final String name) {
         this(null, scope, new NodeList<>(), new SimpleName(name), new NodeList<>());
     }
 
+    public MethodCallExpr(final Expression scope, final SimpleName name) {
```

Comment:
Ever since I put every name and identifier into a node, I've been trying to give `String` shortcuts to reduce the pain.